llvm/test/tools/sycl-post-link/emit_imported_symbols.ll

```diff
@@ -0,0 +1,67 @@
+; This test checks that the -emit-imported-symbols option generates a list of imported symbols
+;
+; RUN: sycl-post-link -symbols -emit-imported-symbols -split=kernel -S < %s -o %t.table
+
+; RUN: FileCheck %s -input-file=%t_0.sym --check-prefixes CHECK-SYM-0
+; RUN: FileCheck %s -input-file=%t_1.sym --check-prefixes CHECK-SYM-1
+; RUN: FileCheck %s -input-file=%t_2.sym --check-prefixes CHECK-SYM-2
+
+;; Function names were chosen so that no function with a 'inside' in their function name is imported
+; RUN: FileCheck %s -input-file=%t_0.prop --check-prefixes CHECK-IMPORTED-SYM-0 --implicit-check-not='inside'
+; RUN: FileCheck %s -input-file=%t_1.prop --check-prefixes CHECK-IMPORTED-SYM-1 --implicit-check-not='inside'
+; RUN: FileCheck %s -input-file=%t_2.prop --check-prefixes CHECK-IMPORTED-SYM-2 --implicit-check-not='inside'
+
+; CHECK-SYM-0: middle
+; CHECK-IMPORTED-SYM-0: [SYCL/imported symbols]
+; CHECK-IMPORTED-SYM-0-NEXT: childD
+
+; CHECK-SYM-1: foo
+; CHECK-IMPORTED-SYM-1: [SYCL/imported symbols]
+; CHECK-IMPORTED-SYM-1-NEXT: childA
+; CHECK-IMPORTED-SYM-1-NEXT: childC
+; CHECK-IMPORTED-SYM-1-NEXT: childD
+
+; CHECK-SYM-2: bar
+; CHECK-IMPORTED-SYM-2: [SYCL/imported symbols]
+; CHECK-IMPORTED-SYM-2-NEXT: childB
+; CHECK-IMPORTED-SYM-2-NEXT: childC
+; CHECK-IMPORTED-SYM-2-NEXT: childD
+; CHECK-IMPORTED-SYM-2-NEXT: _Z7outsidev
+
+target triple = "spir64-unknown-unknown"
+
+; CHECK-NOT: llvm.used
```

Comment:
Why do we need to check for this? Again trying to understand relevance of this check to this PR. Thanks

Reply:
CHECK-NOT has been removed.  It was an oversight due to using a previous testcase as a starting point.